cypress/support/commands.js

```diff
@@ -252,13 +252,13 @@ Cypress.Commands.add('enablePrettyNames', () => {
   cy.__waitForSettingsButton__();
 
   // Visit the settings panel
-  cy.get('[data-test="Change the settings flags"]').click();
+  cy.get('[data-test="global-toolbar-settings-btn"]').click();
 
   // Enable the pretty names setting
-  cy.get('[data-test="pipeline-toggle-input-isPrettyName"]').check({ force: true });
+  cy.get('[data-test*="settings-modal-toggle-isPrettyName-"]').check({ force: true });
```

Comment:
is trailing hyphen part of the convention ? - settings-modal-toggle-isPrettyName**-**